packages/xrpl/test/models/escrowFinish.test.ts

```diff
@@ -51,7 +58,7 @@ describe('EscrowFinish', function () {
   })
 
   it(`throws w/ invalid OfferSequence`, function () {
-    escrow.OfferSequence = '10'
+    escrow.OfferSequence = 'ten'
```

Comment:
nit: I'd use a different string like `abcd` instead of `ten` because I can totally see myself reading this and accidentally internally thinking "10" while I look for the problem elsewhere hahaha